src/App.css

```diff
@@ -1,38 +1,173 @@
-.App {
-  text-align: center;
+html {
+  background-image: radial-gradient(
+    circle at top right,
+    hsl(112, 60%, 23%),
+    hsl(226, 49%, 26%)
+  );
+  min-height: 100%;
+  height: auto;
+  overflow-x: hidden;
+  top: 0;
+  left: 0;
 }
```

Comment:
by doing this you are updating the css on all the pages in the project, i would recommend adding all the css from the html and move it to a class and using it in all the pages you use it